src/app/core/services/error-message.service.ts

```diff
@@ -64,6 +64,10 @@ export class ErrorMessageService {
     return 'Unable to open this issue in Browser';
   }
 
+  public static unableToOpenMilestoneInBrowserMessage() {
+    return 'Unable to open this milestone in Browser';
+  }
```

Comment:
Would it be better to indicate a more appropriate message to users? I believe something like "Milestone is not assigned" would be more appropriate since in the case of milestones the root cause of the issue is the unassigned milestone.

Reply:
Thank you for the suggestion. Updated the error message to `Milestone unassigned`

![image](https://github.com/user-attachments/assets/16ebd904-e959-47e6-9516-a72f3fd9d135)